<?xml version="1.0"?>

<launch>

  <!-- ============================ GOAL ADAPTER =============================== -->

  <!-- $ roslaunch dawn_ik horti_fake.launch -->
  <!-- rosrun relaxed_ik_ros1 rviz_viewer.py -->
  <!-- Copy the values here and dont forget to add commas -->
  <!-- These values are for lite6 -->
  <arg name="init_trans" default="[8.69986108e-02, -7.13350537e-07,  1.54199361e-01]"/>
  <arg name="init_rot" default="[1.83661604e-06,  1.00000000e+00,  5.50981440e-06, -3.67319498e-06]"/>

  <arg name="world_frame" default="world"/>
  <arg name="base_frame" default="head_link_base"/>
  <arg name="eef_frame" default="head_link_eef"/>

  <node name="collision_ik_goal_adapter" pkg="dawn_ik" type="collision_ik_goal_adapter.py" output="screen">
    <param name="init_trans" value="$(arg init_trans)"/>
    <param name="init_rot" value="$(arg init_rot)"/>
    <param name="world_frame" value="$(arg world_frame)"/>
    <param name="base_frame" value="$(arg base_frame)"/>
    <param name="eef_frame" value="$(arg eef_frame)"/>
  </node>

  <!-- ============================ COLLISION ADAPTER =============================== -->

</launch>
